test/PerformanceTests/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+
+
+
+
+
+set(test_targets ${test_targets} PARENT_SCOPE)
```

Comment:
@CiaranWelsh : Do you want the OldData pngs checked in?  If so that's fine; just wanted to be sure you wanted 'em.

Reply:
Nope, actually I might get rid of most of these python tests as I'm currently working on a better way by wrapping vtune CLI commands and running them with python. It remains to be seen whether we can get this running on Azure, but at least we've automated a tool that specializes in performance tests. I'm thinking a new Azure pipeline to sit alongside our main pipeline might work well, though my credentials problem still exists.